jinja2/sandbox.py

```diff
@@ -14,9 +14,17 @@
 """
 import types
 import operator
+from collections import Mapping
 from jinja2.environment import Environment
 from jinja2.exceptions import SecurityError
-from jinja2._compat import string_types, PY2
+from jinja2._compat import string_types, text_type, PY2
+from jinja2.utils import Markup
+
+has_format = False
+if hasattr(text_type, 'format'):
```

Comment:
It seems that jinja2 doesn't support any Python version which does not have `str.format()`/`unicode.format()`, hence this check seems redundant.